build.sbt

```diff
@@ -18,7 +18,7 @@ lazy val riddlIdeaPlugin: Project = Root(
     With.build_info,
     With.coverage(90),
     With.aliases,
-    With.riddl(forJS = false, "0.52.1")
+    With.riddl(forJS = false, "0.52.2-13-89c9057a")
```

Comment:
There's an even newer version with better compatibility: 0.52.2-19-c435714c
Same deal, you have to build and publishLocal yourself

Reply:
Using `publishLocal` seems like a stopgap solution for intermediate versions  used before the plugin hits production - I don't think we want to publish the plugin in Intellij's store including libs generated locally? In which case, this line is irrelevant until we hit marketplace

Reply:
Correct. But 0.52.2.-13-89c9057a as above isn't published either so you're already in that situation If you weren't I wouldn't have mentioned it.  I will be doing a RIDDL release, probably 0.53.0 in the next couple of weeks when I catch my breath!